packages/node_modules/@webex/internal-plugin-devices/test/integration/spec/devices.js

```diff
@@ -0,0 +1,955 @@
+import '@webex/internal-plugin-devices';
+
+import {assert} from '@webex/test-helper-chai';
+import sinon from 'sinon';
+import testUsers from '@webex/test-helper-test-users';
+import WebexCore, {WebexHttpError} from '@webex/webex-core';
+
+
+describe('plugin-wdm', () => {
+  describe('Devices', () => {
+    let webex;
+    let devices;
+    let user;
+
+    beforeEach('create users', () => testUsers.create({count: 1})
```

Comment:
Would sharing a set of test users for all the tests be possible? Otherwise, this may cause the test time to increase significantly

Reply:
Will address this.